_data/companies.yml

```diff
@@ -1321,13 +1321,6 @@
   events: Restricted
   last_update: 2020-03-11
 
-- name: Tripadvisor
-  wfh: Encouraged
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
-
```

Comment:
Looks like your git client is trying to remove tripadvisor. Would you mind fixing that?